From c62b7e5c262b4e2f8322108219e9c35d3a1dac16 Mon Sep 17 00:00:00 2001
From: "Nagamani Chennuboina nagamani.chennuboina@intel.com"
 <nagamani.chennuboina@intel.com>
Date: Mon, 10 Jan 2022 09:44:52 +0530
Subject: [PATCH] Changes to compile xbyak project for openvino

- Added Android.bp and included required file paths

Signed-off-by: Nagamani Chennuboina <nagamani.chennuboina@intel.com>
---
 Android.bp | 9 +++++++++
 1 file changed, 9 insertions(+)
 create mode 100644 Android.bp

diff --git a/Android.bp b/Android.bp
new file mode 100644
index 000000000000..d5d44181574a
--- /dev/null
+++ b/Android.bp
@@ -0,0 +1,9 @@
+//#################################################
+cc_library_headers {
+    name: "xbyak_headers",
+    vendor: true,
+    export_include_dirs: [
+        "gen",
+        "xbyak"
+    ],
+}
-- 
2.33.0

